config-model/src/test/java/com/yahoo/vespa/model/container/ml/ModelsEvaluatorTest.java

```diff
@@ -18,8 +18,7 @@
 public class ModelsEvaluatorTest {
 
     @Test
-    public void testModelsEvaluatorTester() {
-        assumeTrue(OnnxEvaluator.isRuntimeAvailable());
```

Comment:
This is used so that test is not run on arm64, where we have no support for onnx at the moment

Reply:
Thanks. Locally, the assumption failed but the test worked ...